app/views/shared/_footer.html.erb

```diff
@@ -0,0 +1,10 @@
+<footer class="footer">
+  <div class="footer__navigation--container">
```

Comment:
What's the method behind the class names here? If it's BEM, these footer ones aren't quite right (container, links, link and logo are elements, not modifiers)

Reply:
That was copied from a different project, I'll remove the classes and add our own in future